test/metabase/models/card_test.clj

```diff
@@ -775,3 +776,13 @@
                         Dashboard
                         before
                         changes)))))))))
+
+(deftest storing-metabase-version
+  (testing "Newly created Card should know a Metabase version used to create it"
+    (let [card-name (mt/random-name)]
+      (t2/insert! :model/Card (-> (t2.with-temp/with-temp-defaults :model/Card)
+                                  (assoc :name card-name)))
+      (let [card (t2/select-one :model/Card :name card-name)]
+        (is (= (:metabase_version card)
+               config/mb-version-string)))
+      (t2/delete! :model/Card :name card-name))))
```

Comment:
should we have a test for the `update` case too?

Reply:
Makes sense, I've also added a check that it's stored in revision.